src/main/java/com/shapesecurity/shift/reducer/CloneReducer.java

```diff
@@ -131,7 +131,7 @@ public Expression reduceCompoundAssignmentExpression(@NotNull CompoundAssignment
     @NotNull
     @Override
     public MemberExpression reduceComputedMemberExpression(@NotNull ComputedMemberExpression node, @NotNull Node object, @NotNull Node expression) {
-        return new ComputedMemberExpression((Expression) object, (ExpressionSuper) expression);
+        return new ComputedMemberExpression((Expression) expression, (ExpressionSuper) object);
```

Comment:
[Disagree](https://github.com/shapesecurity/shift-java/blob/d3c090ef2de9ba42a3f45fe0aa57aab1123827cc/generated-sources/main/java/com/shapesecurity/shift/ast/ComputedMemberExpression.java#L34). This was only working by accident.